lib/WebService/Stripe.pm

```diff
@@ -76,6 +83,11 @@ method refund_charge(Str $id, HashRef :$data, :$headers) {
     return $self->post( "/v1/charges/$id/refunds", $data, headers => $headers );
 }
 
+method add_source(HashRef|Str $cust, HashRef $data, :$headers) {
```

Comment:
I'm just sticking with Stripe terminology - they use [the word source](https://stripe.com/docs/api#create_charge).